arlas-tests/src/test/java/io/arlas/server/rest/explore/AbstractDescribeTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.rest.explore;
+import io.arlas.server.AbstractTestWithCollection;
+import io.arlas.server.DataSetTool;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ValidatableResponse;
+import org.junit.Test;
+import java.util.Optional;
+import static org.hamcrest.Matchers.equalTo;
+import static io.restassured.RestAssured.given;
+
+public abstract class AbstractDescribeTest extends AbstractTestWithCollection {
+
+    private static final String PARAMS_FIELD_EXCEPT_DUBLIN_CORE = "['index_name', 'type_name', 'id_path', 'geometry_path', 'centroid_path', 'timestamp_path', 'exclude_fields', 'update_max_hits', 'taggable_fields', " +
+            "'exclude_wfs_fields', 'custom_params', 'inspire', 'raster_tile_url', 'raster_tile_width', 'raster_tile_height', 'filter']";
+    private static final String DUBLIN_CORE_FIELDS_EXCEPT_DATE = "['title', 'creator', 'subject', 'description', 'publisher', 'contributor', 'type', 'format', 'identifier', 'source', 'language', 'bbox', 'coverage', " +
+            "'coverage_centroid']";
+
+    @Test
+    public void testDescribeFeature() throws Exception {
+        handleResponse(get(Optional.empty()), new JsonPath(this.getClass().getClassLoader().getResourceAsStream(getDescribeResultPath(DataSetTool.ALIASED_COLLECTION))));
+    }
+
+    /**
+     * Path to expected `describe` result
+     * @return
+     * @param aliasedCollection
+     */
+    protected abstract String getDescribeResultPath(boolean aliasedCollection);
+
+    /**
+     * Compare the response to the given json
+     * @param response
+     * @param jsonPath
+     */
+    protected abstract void handleResponse(ValidatableResponse response, JsonPath jsonPath);
+
+    private ValidatableResponse get(Optional<String> columnFilter) {
+        return given()
+                .header("column-filter", columnFilter.orElse(""))
+                .when()
+                .get(getUrlPath("geodata"))
+                .then();
+    }
+
+    /**
```

Comment:
did you mean `because` instead of before ?

Reply:
You're right, I fixed it